metadata_service/exceptions/exceptions.py

```diff
@@ -3,7 +3,21 @@ def __init__(self, msg):
         super().__init__()
         self.message = {
             'type': 'DATA_NOT_FOUND',
-            'code': 105,
+            'code': 404,
```

Comment:
This has to be 105 as it is specified in the contract here: https://gitlab.sikt.no/raird/data-store-api-doc#error-responses

Reply:
was not aware of this! Will fix 👍🏻